src/components/expandableSection/index.tsx

```diff
@@ -35,11 +35,19 @@ export type ExpandableSectionProps = {
 function ExpandableSection(props: ExpandableSectionProps) {
   const {expanded, sectionHeader, children, top} = props;
 
+  const animate = () => {
+    LayoutAnimation.configureNext({...LayoutAnimation.Presets.easeInEaseOut, duration: 300});
```

Comment:
If you add a log here you'll see two issues:
1. You have an animation on load, you can solve that by using `useDidUpdate` instead of `useEffect`.
2. You have two animations when pressing the header, I'm not sure this is a usage issue (the screen) or the component.

Reply:
I feel like this duplicity is indicating a problem with the API, we should have had either the `onPress` or the `expanded` :/ 
Regarding the new layout animation, make sure you test Android, there was an issue with it last time I've tested it (pre release of 2.3).